src/ontology/edits/oeo-physical.omn

```diff
@@ -3213,6 +3221,120 @@ Class: OEO_00000449
         <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000001
     
     
+Class: OEO_00010000
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It is a gas and can act as an air pollutant. Synonyms are trihydridonitrogen and nitrogen trihydride.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "NH3",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/263
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/356",
+        rdfs:label "ammonia"@de
+    
+    SubClassOf: 
+        OEO_00000331,
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00000055,
+        has_normal_state_of_matter value OEO_00000182
+    
+    
+Class: OEO_00010001
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Carbon monoxide is a portion of matter with the chemical formula CO. It is a gas and can act as an air pollutant.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Carbon monoxide is a portion of matter with the chemical formula CO. It can act as an air pollutant.",
```
Defining it as a *gas* not precise and we refer to the *normal* state of matter anyway.

Reply:
I oriented myself how we did with carbon dioxide:
_Carbon dioxide is a portion of matter with the chemical formula CO2. **It is a gas** that occurs naturally in the atmosphere as a trace gas and can work as a greenhouse gas._

So we should change that, too?

Reply:
Ok. But since it is already there, I'd prefer to take it out.
For C02 we could just write *It occurs naturally in the atmosphere as a trace gas and can work as a greenhouse gas.*  So, *gas* is still there twice in this sentence... 😄

Reply:
@akleinau : Do you agree that _it is a gas_ should be taken out everywhere?

Reply:
I think so. It could lead to confusion. But sth like "is gaseous" is ok of course

Reply:
We could also write everywhere _it has a gaseous normal state of matter_ instead. Then we describe only the property and the definition maps nicely to the `has_normal_state_of_matter` relationship.

Reply:
@stap-m @a-kleinau: What about my last suggestion?

Reply:
yes won't hurt. Maybe would be doubled when it's described like CO2 as a trace gas but there's nothing else speaking against it.

Reply:
So why just for gases? I'd still leave it out of the def, but if you'd like to keep it, it is ok for me.